contrib/golang/filters/go/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/envoyproxy/envoy/contrib/golang/filters/go
```

Comment:
could you add this `go.mod` file to `.github/dependabot.yml` please

Reply:
ok, I'll do this soon, together with the docs.

Reply:
we already move go.mod to the root directory now, https://github.com/envoyproxy/envoy/blob/main/go.mod

Reply:
done